package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-material-components-web",
-  "version": "0.0.33",
+  "version": "0.0.34",
```

Comment:
I don't understand why that would be any more desirable here than all the other times we've bumped the version number in this repo within a single PR.